README.md

```diff
@@ -0,0 +1,84 @@
+  cloud_config  = "${data.template_file.container_instance_cloud_config.rendered}"
+
+  health_check_grace_period = "600"
+  desired_capacity          = "1"
+  min_size                  = "0"
+  max_size                  = "1"
+
+  enabled_metrics = [
+    "GroupMinSize",
+    "GroupMaxSize",
+    "GroupDesiredCapacity",
+    "GroupInServiceInstances",
+    "GroupPendingInstances",
+    "GroupStandbyInstances",
+    "GroupTerminatingInstances",
+    "GroupTotalInstances",
+  ]
+
+  private_subnet_ids = [...]
+
+  project     = "Something"
+  environment = "Staging"
+}
+```
+
+## Variables
+
+- `vpc_id` - ID of VPC meant to house cluster
+- `ami_id` - Cluster instance Amazon Machine Image (AMI) ID
+- `instance_type` - Instance type for cluster instances (default: `t2.micro`)
+- `key_name` - EC2 Key pair name
+- `cloud_config` - `cloud-config` user data supplied to launch configuration for cluster nodes
+- `health_check_grace_period` - Time in seconds after container instance comes into service before checking health (default: `600`)
+- `desired_capacity` - Number of EC2 instances that should be running in cluster (default: `1`)
+- `min_size` - Minimum number of EC2 instances in cluster (default: `0`)
+- `max_size` - Maximum number of EC2 instances in cluster (default: `1`)
+- `enabled_metrics` - A list of metrics to gather for the cluster
+- `private_subnet_ids` - A list of private subnet IDs to launch cluster instances
+- `scale_up_cooldown_seconds` - Number of seconds before allowing another scale up activity (default: `300`)
+- `scale_down_cooldown_seconds` - Number of seconds before allowing another scale down activity (default: `300`)
+- `high_cpu_evaluation_periods` - Number of evaluation periods for high CPU alarm (default: `2`)
+- `high_cpu_period_seconds` - Number of seconds in an evaluation period for high CPU alarm (default: `300`)
+- `high_cpu_threshold_percent` - Threshold as a percentage for high CPU alarm (default: `90`)
+- `low_cpu_evaluation_periods` - Number of evaluation periods for low CPU alarm (default: `2`)
+- `low_cpu_period_seconds` - Number of seconds in an evaluation period for low CPU alarm (default: `300`)
+- `low_cpu_threshold_percent` - Threshold as a percentage for low CPU alarm (default: `10`)
+- `high_memory_evaluation_periods` - Number of evaluation periods for high memory alarm (default: `2`)
+- `high_memory_period_seconds` - Number of seconds in an evaluation period for high memory alarm (default: `300`)
+- `high_memory_threshold_percent` - Threshold as a percentage for high memory alarm (default: `90`)
+- `low_memory_evaluation_periods` - Number of evaluation periods for low memory alarm (default: `2`)
+- `low_memory_period_seconds` - Number of seconds in an evaluation period for low memory alarm (default: `300`)
+- `low_memory_threshold_percent` - Threshold as a percentage for low memory alarm (default: `10`)
+- `ecs_for_ec2_service_role_policy_arn` - IAM policy service role ARN for cluster EC2 instances
```

Comment:
Agreed. I think I left this to keep the door open in case it changed, but it may be safer to hard code it as a constant.